openapi/data_repository_service.swagger.yaml

```diff
@@ -192,144 +152,156 @@ definitions:
           etag              # multipart uploads to blob stores
           sha256
           sha512
-  Bundle:
+  Object:
     type: object
-    required: ['id', 'size', 'created', 'checksums', 'contents']
+    required: ['id']
     properties:
       id:
         type: string
-        description: >-
-          An identifier unique to this Data Bundle.
+        description: |-
+          An identifier unique to this `Object`.
       name:
         type: string
-        description: >-
-          A string that can be used to name a Data Bundle.
-      size:
+        description: |-
+          A string that can be used to name an `Object`.
+      type:
```

Comment:
`type` is a keyword in Swagger and JSON Schema (and `id` is a keyword in JSON Schema). Some of our other APIs might be guilty of this as well, but I wonder if we should try harder to avoid keywords in our property names (e.g., could use `object_id` and `object_type`).

Will open a separate issue for discussion.